src/platform.hpp

```
@@ -39,6 +41,7 @@ struct ebpf_platform_t {
     ebpf_get_program_type_fn get_program_type;
     ebpf_get_helper_prototype_fn get_helper_prototype;
     ebpf_is_helper_usable_fn is_helper_usable;
+    ebpf_get_helper_index_fn get_helper_index;
```

Clang (14.0.6)

[warning] member variable 'get_helper_index' has public visibility

(misc-non-private-member-variables-in-classes)